.travis.yml

```diff
@@ -233,8 +233,8 @@ install:
 script:
   # Now it should be safe to point TMPDIR to a "tricky" setup just for the purpose of testing
   - if [ -n "${_DL_TMPDIR:-}" ]; then export TMPDIR="${_DL_TMPDIR}"; fi
-  # Test installation system-wide
```

Comment:
I wish I recorded in e64286864f38883b119532fe37e906efe312c7a9 **why** we wanted it system-wide -- there must have been a reason. But since forgotten -- oh well, let's proceed with user-wide ;)